lib/actions/bulkUpdateObjects.js

```diff
@@ -0,0 +1,18 @@
+const { messages } = require('elasticio-node');
+const SugarCrm = require('../sugarcrm');
+
+exports.process = async function ProcessAction(msg, cfg) {
+  if (!msg.body.massupdate_params || !Array.isArray(msg.body.massupdate_params.uid)) {
+    this.logger.error('Input "massupdate_params.uid" list isn\'t an Array');
+    this.emit('data', messages.newMessageWithBody({ }));
```

Comment:
Why it doesn't throw an error (or emit error) instead of emitting data (empty object)?
Is it a specific behavior? Should it be documented additionally, because it is not a common-based approach for elastic.io-components.

Reply:
Right. The same as a previous.